frontend/src/components/BackLink/index.js

```diff
@@ -3,18 +3,18 @@ import styles from "./index.module.scss";
 
 import { ReactComponent as Chevron } from "assets/svgs/wheel-background/chevron.svg";
 
-const BackLink = ({ href }) => {
+const BackLink = ({ handleReturn }) => {
   return (
     <div className={styles.backLinkWrapper}>
-      <a href={href} className={styles.backLink}>
+      <button onClick={handleReturn} className={styles.backLink}>
```

Comment:
Oe. It's late and I'm blanking. But `handleReturn` reminds me too much of `return` ....